Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -68,6 +68,7 @@ span.DropDownMenuSelect a.DropDownMenuButton
 span.DropDownMenuSelect:not(.disabled) a.DropDownMenuLabel
 {
   color: var(--color-standard-text);
+  font-family: var(--remotion-themed-font-bold);
```

Comment:
Because of the `:not(...)` selector the font is not applied to disabled controls